django/test/client.py

```diff
@@ -815,6 +859,20 @@ def logout(self):
         logout(request)
         self.cookies = SimpleCookie()
 
+    async def alogout(self):
+        """Log out the user by removing the cookies and session object."""
+        from django.contrib.auth import aget_user, alogout
+
+        request = HttpRequest()
+        if self.session:
```

Comment:
How come not `session = await self.asession()`?

Reply:
oops!